tests/regression/nextcloud-rule-exclusions-plugin/9508400.yaml

```diff
@@ -67,11 +68,50 @@ tests:
               Host: localhost
               User-Agent: OWASP CRS test agent
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+              Content-Type: application/json
             port: 80
             method: POST
-            uri: /index.php/login
-            data: "redirect_url=/login/v2/grant?user=Esad%20Cetiner&stateToken=sample-token"
+            uri: /login/confirm
+            data: |
+              {"password": "<script>"}
+            version: HTTP/1.1
+          output:
+            no_log_contains: id "941101"
+  - test_title: 9508400-5
+    desc: Disable OWASP CRS for password when an admin is reauthenticating without pretty URLs
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Host: localhost
+              User-Agent: OWASP CRS test agent
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+              Content-Type: application/json
+            port: 80
+            method: POST
+            uri: /index.php/login/confirm
+            data: |
+              {"password": "<script>"}
             version: HTTP/1.1
           output:
-            no_log_contains: |-
-              id "920230"|id "932190"|id "932200"|id "942431"|id "942432"
+            no_log_contains: id "941101"
+  - test_title: 9508400-6
```

Comment:
This test looks like a duplicate of the test above.

Reply:
I'll remove the tests that check without pretty URLs, they are mostly redundant (I think pretty URLs are the default now in Nextcloud anyways). These 2 tests were meant to check both with and without pretty URLs